tooling/noirc_abi_wasm/package.json

```diff
@@ -21,7 +21,6 @@
     "type": "git",
     "url": "https://github.com/noir-lang/noir.git"
   },
-  "packageManager": "yarn@3.5.1",
```

Comment:
why is this removed?

Reply:
This is defined at the workspace root now. These are leftovers from when we didn't have a workspace.

Reply:
Ah we have this in two other crates I think -- I'll push another PR to remove those